### PR TITLE
Bot core

### PR DESCRIPTION
**Pull request by mcamurri**
This merge request replaces the locally build `ins_t` message with the one provided by `bot_core` package.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -7,15 +7,14 @@ include(cmake/pods.cmake)
...
 add_executable(microstrain src/microstrain_comm.cpp)
-pods_use_pkg_config_packages(microstrain bot2-core lcmtypes_microstrain bot2-param-client)
+pods_use_pkg_config_packages(microstrain bot2-core bot2-param-client)
 pods_install_executables(microstrain)
 
 
 # pass through microstrain quaternion as a bot_core_pose_t (for simple visualization)
 add_executable(microstrain_pose src/microstrain_pose.cpp)
-pods_use_pkg_config_packages(microstrain_pose lcmtypes_microstrain)
+pods_use_pkg_config_packages(microstrain_pose bot2-core)
```
Comment (mauricefallon) [inline CMakeLists.txt:19]: Is lcmtypes_bot2-core sufficient?

Comment (mcamurri) [inline CMakeLists.txt:19]: Yes, maybe. I'm trying to compile now, to see.

Comment (mcamurri) [inline CMakeLists.txt:19]: Locally it compiles with `lcmtypes_bot2-core`.  

Globally...there's a problem: there is a discrete amount of other packages which depends on `microstrain_lcmtypes`, especially the ones related to  `gpf-laser-lib`

Watch https://github.com/openhumanoids/pronto/pull/61/commits/4e0b48c9e63a3fa87f9f7fc61691f2df8b51b4ab

**Comment by mauricefallon** [comment]
Thanks @mcamurri , I'll deal with the related issues in due course. I've made an issue internally within OH

**Pull request merged**